src/tests/gov/nasa/jpf/test/java/io/FileTest.java

```diff
@@ -102,5 +104,19 @@ public void testEquals() {
       assert file.equals(new Object()) == false;
     }
   }
-  
+
+  @Test
+  public void testToURI(){
+    File file = new File("testfile.txt");
+    URI expectedURI = null;
+    try {
+      expectedURI = new URI("file:testfile.txt");
+    } catch (URISyntaxException e) {
+      fail("URISyntaxException thrown while constructing expected URI");
+    }
+
+    URI actualURI = file.toURI();
+    assertNotNull("URI should not be null",actualURI);
```

Comment:
The non-`null` check is subsumed by the assertion below, so you can remove this line.

Reply:
Okay will remove it.